Sources/Containerization/SandboxContext/SandboxContext.proto

```diff
@@ -148,8 +148,9 @@ message CreateProcessRequest {
   optional uint32 stdin = 3;
   optional uint32 stdout = 4;
   optional uint32 stderr = 5;
-  bytes configuration = 6;
-  optional bytes options = 7;
+  optional string ociRuntimePath = 6;
```

Comment:
should this be added to the end?

Reply:
It could, but I liked this order, and we're free to break anything as it stands now